hw2/05-form.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>05 - Form</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+    <script defer src="05-form.js"></script>
+  </head>
+  <body class="bg-dark">
+    <main class="bg-light mx-auto w-50 mt-5 rounded p-3">
+      <form>
+        <h1 class="text-center my-3">Exercise 05 - Form</h1>
+        <div class="mb-3 mt-5">
+          <label for="fullname" class="form-label fw-bold">Full Name:</label>
+          <input type="text" class="form-control" id="fullname">
+        </div>
+        <div class="mb-3">
+          <label for="email" class="form-label fw-bold">Email:</label>
+          <input type="email" class="form-control" id="email">
+        </div>
+        <div class="mb-3">
+          <label for="status" class="form-label fw-bold">Registration Status:</label>
+          <select class="form-select" aria-label="Registration Status">
+            <option selected>Choose an Option</option>
+            <option value="Registered">Registered</option>
+            <option value="Waitlisted">Waitlisted</option>
+            <option value="Drop">Drop</option>
+          </select>
+        </div>
+        <div class="mb-3">
```

Comment:
Fieldset element can  be used here to avoid WAVE alert. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset